Automattic/jetpack
# Masterbar: Remove unused RTL_Admin_Bar class

It looks like `RTL_Admin_Bar` has been defunct since #11088 moved the initialization to `admin_bar_init`.

Fixes #17773.

#### Changes proposed in this Pull Request:

* Removes `RTL_Admin_Bar` class.
* Removes filter callback and hook that activated the class.

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* Navigate to https://wordpress.com/me/account and select a RTL language like Hebrew.
* In `Jetpack::get_connected_user_data()` comment out the line that returns `$cached_user_data`.
* On your Jetpack site under Settings > General, set your site language to a RTL language like Hebrew.
* Reload wp-admin and make sure the masterbar renders in RTL.

#### Proposed changelog entry for your changes:

* Removed unused class `RTL_Admin_Bar`.
